src/main/java/fr/inria/diversify/mutant/pit/GradlePitTaskAndOptions.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.diversify.mutant.pit;
+
+/**
+ * Created by Daniele Gagliardi
+ * daniele.gagliardi@eng.it
+ * on 21/07/17.
+ */
+
+public class GradlePitTaskAndOptions {
+
+    public static boolean descartesMode = false;
+
+    public static boolean evosuiteMode = false;
+
+    public static final String OPT_WITH_HISTORY = "enableDefaultIncrementalAnalysis = ";
+
+    public static final String OPT_VALUE_REPORT_DIR = "reportDir = 'build/pit-reports'";
+
+    public static final String OPT_TARGET_CLASSES = "targetClasses = ";
+
+    public static final String OPT_VALUE_FORMAT = "outputFormats = ['CSV','HTML']";
+
+    public static final String OPT_TARGET_TESTS = "targetTests = ";
+
+    public static final String PROPERTY_ADDITIONAL_CP_ELEMENTS = "additionalMutableCodePaths = ";
+
+    public static final String PROPERTY_EXCLUDED_CLASSES = "excludedClasses = ";
+
+    //Apparently Unsupported by PIT plugin
+    public static final String OPT_ADDITIONAL_CP_ELEMENTS = "classPath = ";
```

Comment:
Hi @danglotb , I've used the two parameters as you suggested, and now it works! Thanks. IN the pitest gradle plugin page (http://gradle-pitest-plugin.solidsoft.info/) they say " All the command line options are supported", and the option about timeout was named timeoutConst. We can pass them as command line options, or otherwise through the properties file: what do you think?